packages/dsh-tauri/src/client/service/invoke.ts

```
@@ -0,0 +1,99 @@
+ *
+ * 协议（与宿主监听器逐字一致）：
+ *   iframe → 宿主：{ source: 'dsh-tauri-invoke', type: 'dsh://tauri:invoke',
+ *                     cmd, args, nonce }
+ *   宿主 → iframe：{ source: 'dsh-desktop-invoke', type: 'dsh://tauri:reply',
+ *                     nonce, ok, value | error }
+ *
+ * 可靠性：每条请求带唯一 nonce，应答按 nonce 精确匹配避免串线；等待超时
+ * （INVOKE_TIMEOUT_MS）或局部失败统一 reject，并登记进宿主错误注册表便于排查。
+ */
+import { INVOKE_TIMEOUT_MS, PLUGIN_ID, SRC_INVOKE, SRC_INVOKE_REPLY, TYPE_INVOKE, TYPE_INVOKE_REPLY } from '../constants'
+import type { InvokeBridgeRequest, InvokeBridgeReply } from '../types'
```

ESLint

[error] Expected "../types" (type-parent) to come before "../constants" (value-parent).

(perfectionist/sort-imports)